OpenSourcePolitics/decidim-module-transparent_trash
# [Feature] - Disable access to versioning for illegal initiatives

#### Description 

Initiatives with state `illegal` must have hidden content (title, description). At the moment users and not logged in users can retrieve original content through the initiative's versioning.

This PR add permissions to block access to illegal initiatives versioning unless user is author or admin.

#### How to test

* Create new initiative
* Change status to Illegal

As a non logged in user, or a regular user : 
* Try to access illegal initiative versioning
* **should be unauthorized to access index or show view for a version**

As an author or admin : 
* Try to access illegal initiative versioning
* **Should be able to access index and show view with original content**
